### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by ciangalligan**
Have tested on master, should work :)
Let me know if a link is wrong or something doesn't match up

### CODE REVIEW

File: src/components/Nav.js [hunk 1]
```diff
@@ -15,9 +16,9 @@ const _Nav = () => {
   const user = useContext(UserContext);
   return (
     <div className="flexnone">
-      <Navbar bg="light" variant="light" expand="md">
```
Comment (johnmarksinclair) [inline src/components/Nav.js:18]: have to leave expand="md" so it goes into a dropdown on mobile

File: src/index.css [hunk 2]
```diff
@@ -11,3 +11,11 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
```
Comment (johnmarksinclair) [inline src/index.css:18]: oh this replacing expand-md?

**Review by johnmarksinclair — COMMENTED** [review]
does the nav still collapse on mobile?

File: src/components/Nav.js (hunk 1 above)
Comment (ciangalligan) [reply src/components/Nav.js:18]: You're right! Will fix now, css change was for centring search bar.

**Review by johnmarksinclair — APPROVED** [review]

**Pull request merged**